### PR TITLE
Makes rings give mood boosts instead of dodge chance

### PR DESCRIPTION
**Pull request by ma44**
## About The Pull Request

Makes rings give mood boosts rather than dodge chance

## Why It's Good For The Game

1. For some, the immersion of a ring that allows you to dodge bullets is pretty not immersive in a fallout universe

2. "Shaft miner grinding Lavaland bosses by themselves"; The main method of acquiring this is through mining the minerals,; this entire process is RNG based with varying success based on technology, and for a roleplay server, may not even require interaction with players to be able to craft this; various items like guns and armor crafting at least has more potential for player:tm: interaction.

3. Powergamers doing obsessive mining for the diamonds can definitely lead to mineral bloat leading to crafting inflation because on how much RNG goes into diamonds, or in the case of BoS, just be another day for endlessly printing RND goodies

The idea with this PR is
1. Make the ring a bit more set "in reality" (in a fallout game) with someone many people can agree on; having a nice ring on you gives a mood buff
2. While powergamers may still want to grind for this, there's easier ways of gaining mood and those ways generally involves food or chemical/plant crafting and those methods actually can involve interactions with other players.

## Changelog
:cl: ma44
balance: Rings now give a mood boost instead of straight up dodge chance against projectiles
/:cl:

### CODE REVIEW

File: code/modules/clothing/gloves/ring.dm [hunk 1]
```diff
@@ -3,23 +3,33 @@
...
+/obj/item/clothing/gloves/ring/dropped(mob/user)
+	SEND_SIGNAL(user, COMSIG_CLEAR_MOOD_EVENT, "ringbuff")
+
+/*
+/obj/item/clothing/gloves/ring/proc/newEquipSlot(mob/living/carbon/user, slot)
+	if (slot == SLOT_GLOVES && istype(user))
+		SEND_SIGNAL(user, COMSIG_ADD_MOOD_EVENT, "ringbuff", mood_event_on_equip)
+	else
+		SEND_SIGNAL(user, COMSIG_CLEAR_MOOD_EVENT, "ringbuff")
+*/
```
Comment (Rohesie) [inline code/modules/clothing/gloves/ring.dm:32]:
```suggestion
```

**Comment by thedarkcrusader** [comment]
> ## About The Pull Request
> Makes rings give mood boosts rather than dodge chance
> 
> ## Why It's Good For The Game
> 1. For some, the immersion of a ring that allows you to dodge bullets is pretty not immersive in a fallout universe

Screenshot_20210723-024403_Chrome

Yes you totally cant bullet dodge in fallout its not immersive at all

**Comment by Corinthian555** [comment]
This makes sense and is a reasonable change.

File: code/modules/clothing/gloves/ring.dm [hunk 2]
```diff
@@ -3,23 +3,26 @@
 	desc = "A tiny gold ring, sized to wrap around a finger."
 	gender = NEUTER
 	w_class = WEIGHT_CLASS_TINY
+	obj_flags = UNIQUE_RENAME //Of course it should be renamable!
```
Comment (Rohesie) [inline code/modules/clothing/gloves/ring.dm:6]:
```suggestion
```
Unrelated change.
Better not expand on this particular feature.
